verilog/tools/obfuscator/verilog_obfuscate.cc

```diff
@@ -50,6 +52,12 @@ ABSL_FLAG(                //
     "reverse to de-obfuscate the source code, and do not obfuscate any unseen "
     "identifiers.  There is no need to --save_map with this option, because "
     "no new substitutions are established.");
+ABSL_FLAG(                //
+    bool, preserve_interface, false,  //
```

Comment:
Can you also document this new feature in obfuscator/README.md?  (Run `--help` and copy-paste.)

Reply:
Sure! 😉